content/soarian/r4/clinical/diagnostics/observation.md

```diff
@@ -0,0 +1,186 @@
+  * Effective date time
+  * Effective period
+* [Issued](https://hl7.org/fhir/R4/observation-definitions.html#Observation.issued){:target="_blank"}
+* [Value[x]](https://hl7.org/fhir/R4/observation-definitions.html#Observation.value_x_){:target="_blank"} 
+    * Value codeable concept
+    * Value date time
+    * Value integer
+    * Value range
+    * Value ratio
+    * Value string
+    * Value time
+    * Value quantity
+    * Extension 
+* [Component](https://hl7.org/fhir/R4/observation-definitions.html#Observation.component){:target="_blank"}
+    * [Component code](https://hl7.org/fhir/R4/observation-definitions.html#Observation.component.code){:target="_blank"}
+    * [Component value[x]](https://hl7.org/fhir/R4/observation-definitions.html#Observation.component.value_x_){:target="_blank"}
+        * valueCodeableConcept
+        * valueDateTime
+        * valueInteger
+        * valueString
+        * valueQuantity
+    * [Component data absent reason](https://hl7.org/fhir/R4/observation-definitions.html#Observation.component.dataAbsentReason){:target="_blank"}
+* [Data absent reason](https://hl7.org/fhir/R4/observation-definitions.html#Observation.dataAbsentReason){:target="_blank"}
+* [Interpretation](https://hl7.org/fhir/R4/observation-definitions.html#Observation.interpretation){:target="_blank"}
+* [Note](https://hl7.org/fhir/R4/observation-definitions.html#Observation.note){:target="_blank"}
+* [Member Observations](https://hl7.org/fhir/R4/observation-definitions.html#Observation.hasMember){:target="_blank"}
+* [Derived From](https://hl7.org/fhir/R4/observation-definitions.html#Observation.derivedFrom){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_observation, :r4) %>
+
+### Extensions
+
+   *  valueAttachment: The URL for this extension is defined as: `http://hl7.org/fhir/5.0/StructureDefinition/extension-Observation.valueAttachment` This extension is defined and referenced from the newer version of Fast Healthcare Interoperability Resources<sup>®</sup> (FHIR<sup>®</sup>). See [Extensions for converting between versions](https://www.hl7.org/fhir/r4/versions.html#extensions) and [R5 Snapshot of Observation.value](https://hl7.org/fhir/2020Feb/observation-definitions.html#Observation.value_x_) for more information.
+
+## Search
+
+Search for Observation resources that meet the specified query parameters:
+
+    GET /Observation?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
+
+### Parameters
+
+ Name          | Required?           | Type            | Description
+---------------|---------------------|-----------------|----------------------------------------------------------------------------------------------------------------------------
+ `_id`         | This, or `patient`  | [`token`]       | The logical resource id associated with the resource.
```

Comment:
In the first row of the Description, change "id" to "ID".

PRODOC-160857

Reply:
Changed id to "ID"